engine/particle/wscript

```diff
@@ -30,10 +30,11 @@ def configure(conf):
     conf.recurse('src')
 
     conf.env.append_value('INCLUDES', "default/src")
-    conf.env['STLIB_DLIB_NOASAN']           = ['dlib_noasan']
-    conf.env['STLIB_DDF_NOASAN']            = ['ddf_noasan']
-    conf.env['STLIB_GRAPHICS_NULL']         = ['graphics_null', 'graphics_proto']
-    conf.env['STLIB_GRAPHICS_PROTO_NOASAN'] = ['graphics_proto_noasan']
+    conf.env['STLIB_DLIB_NOASAN']     = ['dlib_noasan']
+    conf.env['STLIB_DDF_NOASAN']      = ['ddf_noasan']
+    conf.env['STLIB_GRAPHICS_NULL']   = ['graphics_null', 'graphics_proto']
+    conf.env['STLIB_GRAPHICS_NOASAN'] = ['graphics', 'graphics_proto', 'graphics_proto_noasan']
```

Comment:
This looks a bit off. Have you verified with `--with-asan`?
1) neither graphics or graphics_proto are using the "noasan" suffix, which means that it's going to be the "ASAN" linkage used. 
2) graphics_proto and graphics_proto_noasan are the same, just with different linkage?
If so , then it's a little bit random to add two versions of the same library on the command line.